AEPMessaging/Sources/Messaging+PublicAPI.swift

```diff
@@ -64,4 +64,31 @@ import UserNotifications
 
         MobileCore.dispatch(event: event)
     }
+    
+    // MARK: Message Feed
+    
+    /// This API dispatches an event to fetch message feeds for the provided surface paths from the Adobe Journey Optimizer via the Experience Edge network.
+    /// - Parameter surfacePaths: An array of surface path strings
+    static func updateFeedsForSurfacePaths(_ surfacePaths: [String]) {
+        let validSurfacePaths = surfacePaths
+            .filter { !$0.isEmpty }
+        
+        guard !validSurfacePaths.isEmpty else {
+            Log.warning(label: MessagingConstants.LOG_TAG,
+                        "Cannot update feeds as the provided surface paths array is empty, or has one or more empty items.")
+            return
+        }
+
+        let eventData: [String: Any] = [
+            MessagingConstants.Event.Data.Key.UPDATE_FEEDS: true,
```

Comment:
do we need this bool in the eventData?  can the presence of surfaces suffice?

Reply:
As discussed offline we will need to distinguish this event from the event for get feeds API which will contain requested surface paths as well.